### ISSUE TITLE
Midpoint Review - Accessibility Feedback - EDM and VEBT team, GI Bill Comparison Tool homepage

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#102227]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [x] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [x] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [x] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Sorry for another delayed feedback ticket!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] **Should:** Reiterating an item from the design intent, recommending that you explore separate lines of text for headings and links. As discussed in the meeting, this is definitely not a "must" item and isn't currently required by the design system --- and there are a ton of examples across VA.gov that have headings-as-links, including/especially the homepage. But there was some user research for an MHV product in which some screen reader users missed that certain headings were links. That put this on my radar as something to watch for and, when appropriate, recommend exploring alternatives.
   
   I haven't managed to track down the study where this came up, just a Slack conversation about it with one of MHV's accessibility specialists. It was something that came up incidentally in the course of testing a different user interaction with screen reader users, but it's borne out through direct testing with some (but not all) screen readers: When moving through a page skipping heading-to-heading, some screen readers will only announce a heading-as-link as a heading (ie. won't announce that it's also a link). A screen reader user may then have their screen reader announce the content that immediately follows the heading, but never hear anything indicating how to move on to where they want to go without backtracking.
   
   All of this is to say: If you're comfortable moving forward with the current design, this is by no means a blocker. But as you iterate on this design in the future and as you receive feedback from your users, definitely be aware of this as a potential point of friction.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone

**Comment by cassidy-beach** [comment]
Created [VEBT-1368](https://jira.devops.va.gov/browse/VEBT-1368) to address this at a later date
